fpdf/fpdf.py

```diff
@@ -4882,6 +4882,23 @@ def table(self, *args, **kwargs):
         yield table
         table.render()
 
+    def create_table_from_dataframe(self, df, *args, **kwargs):
```

Comment:
Such function should probably live in its own `fpdf/pandas.py` module.

@gmischler already provided a detailed explanation on why this should not be there, in `fpdf2` core source file 😊